src/controllers/auth.controllers.js

```diff
@@ -125,3 +129,8 @@ export const deleteUser = async (req, res, next) => {
     res.send(e);
   }
 };
+
+export const refreshUserToken = (req, res) => {
+  const token = generateToken(req.user);
+  res.status(201).json({ token });
```

Comment:
it not clear what is what here, can you rename it so it is clear what is "refresh token" and what is "access token"

Reply:
refreshtoken is written refresh token everywhere, and when there is token it is the access token

Reply:
this function is kinda confusing, so its called refreshUserToken but then it generates a new access token? can you maybe explain us the exact logic of token refreshing to us?